src/plugins/wb-disable/disable.js

```diff
@@ -67,12 +89,19 @@ var componentName = "wb-disable",
 						// Store preference for WET plugins and polyfills to be enabled in localStorage
 						localStorage.setItem( "wbdisable", "false" );
 					}
+
+					// Remove variable from URL
+					var lc = window.location.href.replace( "wbdisable=false", "" ).replace( "?#", "#" );
+					if ( lc.indexOf( "?" ) === ( lc.length - 1 ) ) {
+						lc = lc.replace( "?", "" );
+					}
+					window.history.replaceState( "", "", lc );
```

Comment:
I do not see any concern here as it the history information of the current page that is updated for an url that do not include "wbdisable=false".